cas-server-documentation/installation/Ignite-Ticket-Registry.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title: CAS - Ignite Ticket Registry
+---
+
+# Ignite Ticket Registry
+Ignite integration is enabled by including the following dependency in the Maven WAR overlay:
+
+{% highlight xml %}
+<dependency>
+     <groupId>org.jasig.cas</groupId>
+     <artifactId>cas-server-integration-ignite</artifactId>
+     <version>${cas.version}</version>
+</dependency>
+{% endhighlight %}
+
+`IgniteTicketRegistry` stores tickets in an [Ignite](http://ignite.apache.org/) instance.
+
+
+## Distributed Cache
+Distributed caches are recommended for HA architectures since they offer fault tolerance in the ticket storage subsystem. The registry
+maintains service tickets and ticket-granting tickets in two separate caches, so that:
+
+* Ticket Granting Tickets remain valid for a long time, replicated asynchronously
+* Service Tickets are short lived and must be replicated right away because the requests
+  to validate them may very likely arrive at different CAS cluster nodes
+
+Enable the registry via:
+
+{% highlight xml %}
+<alias name="igniteTicketRegistry" alias="ticketRegistry" />
+{% endhighlight %}
+
+
+###TLS Replication
+Ignite supports replication over TLS for distributed caches composed of two or more nodes. To learn more about TLS replication with Ignite,
+[see this resource](https://apacheignite.readme.io/docs/ssltls).
+
+Enable TLS via:
+
+{% highlight xml %}
+<bean id="igniteSslContext" class="org.apache.ignite.ssl.SslContextFactory">
+    <property name="keyStoreFilePath" value="keystore/server.jks"/>
+    <property name="keyStorePassword" value="123456"/>
+    <property name="trustStoreFilePath" value="keystore/trust.jks"/>
+    <property name="trustStorePassword" value="123456"/>
+</bean>
+{% endhighlight %}
+
+{% highlight properties %}
+ignite.ssl.context=igniteSslContext
```

Comment:
Ah never mind I see it. I do think it would be better if you don't require adopters to create this bean config, and just have them specify properties so YOU then create this bean for them.